frontend/src/features/moti/mapSideBar/forms/PropertySearchForm.tsx

```diff
@@ -120,7 +124,10 @@ export const PropertySearchForm = ({
               />
             </Form.Row>
             <Form.Row>
-              <Label>Address</Label>
+              <Label>
+                Address&nbsp;
+                <TooltipIcon toolTipId="geocoder-tooltip" toolTip={geocoderTooltip}></TooltipIcon>
```

Comment:
this tooltip tells the user why the geocoder search button is disabled by default.